book/entities.rst

```diff
@@ -0,0 +1,1182 @@
+    | ``priority``      | Defines the order in which options will be shown in grid views and      |
+    |                   | forms (options with a higher priority will be displayed before options  |
+    |                   | with a lower priority).                                                 |
+    +-------------------+-------------------------------------------------------------------------+
+
+``grid``
+    Configures the way the field is presented in a list view:
+
+    +-------------------+-------------------------------------------------------------------------+
+    | Option            | Description                                                             |
+    +===================+=========================================================================+
+    | ``type``          | The attribute type                                                      |
+    +-------------------+-------------------------------------------------------------------------+
+    | ``label``         | The grid column headline                                                |
+    +-------------------+-------------------------------------------------------------------------+
+    | * ``show_filter`` | These options control whether the view can be filtered by the attribute |
+    | * ``filterable``  | value and how the filter options look like.                             |
+    | * ``filter_type`` |                                                                         |
+    +-------------------+-------------------------------------------------------------------------+
+    | ``sortable``      | When enabled, the user can sort the table by clicking on the attribute  |
+    |                   | column's title.                                                         |
+    +-------------------+-------------------------------------------------------------------------+
+
+    .. note::
+
+        In order to use the attribute in a grid view, it
+        :ref:`needs to be indexed <book-entities-indexed-attributes>`.
+
+``form``
+    You use these options to control how the actual value can be configured by the user:
+
+    =================  ==============================================================
+    Option             Description
+    =================  ==============================================================
+    ``type``           The form type
+    ``options``        Additional options controlling the form layout:
+    * ``block``        The block of the form in which the attribute will be displayed
+    * ``label``        The field label
+    * ``choices``      Possible values from which the user can choose one
+    * ``empty_value``  The value that is taken when the user makes no choice
+    =================  ==============================================================
+
+Secondly, you need to update all configurable entities after configuration parameters have been
```

Comment:
unfortunately the `oro:entity-config:update` does not work with modified parameters. Only new values are added to the database. To process modified parameters you have to execute this command with `--force` option